include/swift/Runtime/RuntimeFunctions.def

```diff
@@ -236,6 +236,24 @@ FUNCTION(NativeStrongReleaseN, swift_release_n, C_CC, AlwaysAvailable,
          EFFECT(RefCounting, Deallocating),
          UNKNOWN_MEMEFFECTS)
 
+// void swift_retainMultiple(void **ptr, size_t count)
+FUNCTION(NativeStrongRetainMultiple, swift_retainMultiple, C_CC,
+         RetainReleaseMultipleAvailability,
+         RETURNS(VoidTy),
+         ARGS(Int8PtrTy, SizeTy),
+         ATTRS(NoUnwind),
+         EFFECT(RefCounting),
+         UNKNOWN_MEMEFFECTS)
```

Comment:
Huh. I wonder why retain is unknown memeffects. I get why release is.

(I don't think this is wrong since the others do it, I'm just curious)